src/clj/game/cards/resources.clj

```diff
@@ -2027,6 +2027,35 @@
                                 :pre-breaker-strength {:req (req (= (:cid target)(:cid (:hai-target card))))
                                                        :effect (effect (breaker-strength-bonus 2))}}) card))}}
 
+   "The Nihilist"
+   (let [has-2-virus-tokens? (req (<= 2 (reduce + (map #(get-counters % :virus) (get-all-installed state)))))
```

Comment:
Not that it matters much, but is there any difference between `reduce +` and `+`?

Reply:
Oh, I forgot + can take multiple arguments. Then no!

Reply:
Oh, wait, I also forgot + doesn't take a list. Oops. So `reduce +` could become `apply +`.

Reply:
Well you already have a map, so maybe you want something like `(reduce %(+ %1 (get-counters %2 :virus)) 0 (get-all-installed state))`.

Reply:
Ah, yes, using an existing function seems much better :)